server.js

```diff
@@ -29,6 +71,85 @@ app.get("/", (req, res) => {
   res.send("Hello Technigo!");
 });
 
+// app.get("/songs", async (req, res) => {
+//   const allTheSongs = await Song.find({}) //to find all the songs
+//   res.status(200).json({
+//     success: true,
+//     body: allTheSongs
+//   });
+// });
+
+app.get("/songs/id/:id", async (req, res) => {
```

Comment:
Great work :) Maybe it had been more intuitive to have the endpoint /songs before songs/id/:id. But that is just a minor thing.